TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -424,7 +425,29 @@ ScriptReserved_GetSlotHP, & Moveable::GetSlotHP,
 // @tparam Objects.ObjID ObjectID to take animation and stateID from,
 // @tparam int animNumber animation from object
 // @tparam int stateID state from object
-	ScriptReserved_AnimFromObject, &Moveable::AnimFromObject);
+	ScriptReserved_AnimFromObject, &Moveable::AnimFromObject,
+		
+	/// Get a creature moveable's current enemy
+	// @Function Moveable:GetCreatureEnemy
+	ScriptReserved_GetCreatureEnemy, &Moveable::GetCreatureEnemy,
+
+	/// Clears a creature moveable's current enemy
+	// @Function Moveable:ClearCreatureEnemy
+	ScriptReserved_ClearCreatureEnemy, &Moveable::ClearCreatureEnemy,
+
+	/// Set a creature moveable's current enemy
+	// @Function Moveable:SetCreatureEnemy
+	// @tparam Moveable A moveable for the creature to treat as enemy
+	ScriptReserved_SetCreatureEnemy, &Moveable::SetCreatureEnemy,
+
+	/// Get a creature moveable's target vector
+	// @Function Moveable:GetCreatureTarget
+	// @treturn Vec3 the creature's current target vector
+	ScriptReserved_GetCreatureTarget, &Moveable::GetCreatureTarget,
+
+	/// Set a creature moveable's target vector
```

Comment:
You were missing the function here. Correct it to:

// @function Moveable:SetCreatureTarget
// @tparam Vec3 position for the creature to target